### ISSUE TITLE
ML new Keras layers

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#4605]
Several new layer types are needed required for image processing. The initial new set could be:
- Add
- AveragePooling2D
- BatchNormalization
- Concatenate
- Conv2D
- Conv2DTranspose
- GaussianNoise
- GlobalAveragePooling2D
- MaxPooling2D
- SeparableConv2D
- UpSampling2D
- ZeroPadding2D

Some layers require more than the current 3 values shown in the UI. For example, the Conv2D may require many arguments:
Layer: Conv2D, Dimensionality: 32, Kernel Size: 3, Strides: 2, Padding: same, Activation: relu, Kernel Initializer: he_normal

For convolutional models, we may want to use the full browser width for the model configuration panel.

**Comment by gurhar1133** [comment]
@moellep I think we should save the Add and Concatenate layers for [#4604](https://github.com/radiasoft/sirepo/issues/4604) 

what do you think?

**Comment by moellep** [comment]
Yes, that would be fine to move the Add and Concatenate layers to the related UI issue.
